helpers/bandchain2.js/src/client.ts

```diff
@@ -0,0 +1,55 @@
+import axios from 'axios'
+import { DataSource } from 'data'
+
+export default class Client {
+  rpcUrl: string
+  constructor(rpcUrl: string) {
+    this.rpcUrl = rpcUrl
+  }
+
+  private async get(path: string, ...params: object[]) {
```

Comment:
IMO, you can just pass param as a single object.
Like this
```
private async get(path: string, param?: object){
...
}
```
And use it like this
```
// without param
const response = await this.getResult(`/oracle/data_sources/${id}`)

// with param
const response = await this.getResult(`/oracle/last_request`, {"min_count": 3, "ask_count": 4})
```